utils/imports.php

```diff
@@ -3,6 +3,7 @@
 
 	require_once(dirname(dirname(__FILE__)).'/lib/init-cmd.php');
 	ini_set('memory_limit', '800M');
+	$sDefaultDataPath = CONST_BasePath.'/data/tiger2014';
```

Comment:
How about moving this into settings.php? Then it can be reused in `./utils/setup.php` below. Also, I'd leave out the year by default. It will just cause again confusion next year.